### PR TITLE
Edits made

### PR DESCRIPTION
**Pull request by IamAnkita-create**
some grammartical errors were corrected

### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by ankitasharma88990** [comment]
Please accept as it's the end of hacktoberfest.
Thank you.

File: RespondingToIssuesAndPullRequests.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Each tool has it’s own label, e.g. _fastlane_, `fastlane_core` and _gym_.
     - Repeat the last step for each of the related PRs that the user submitted.
 - After checking out a user’s code, you should always make sure that the tests are still working.
   - Run `bundle install` to make sure all dependencies are installed
-  - Use `bundle exec fastlane test` from the root directory to run all validation steps (tests, rubocop, etc)
```
Comment (att55) [inline RespondingToIssuesAndPullRequests.md]: I think most users type `fastlane` (not initial capital).
And, `rubocop` is correct.
https://github.com/rubocop-hq/rubocop

File: README.md [hunk 2]
```diff
@@ -180,13 +180,13 @@ _fastlane_ tracks a few key metrics to understand how developers are using the t
 * The number of _fastlane_ runs
 * A salted hash of the app identifier or package name, which helps us anonymously identify unique usage of _fastlane_
  
-You can easily opt-out of metrics collection by adding `opt_out_usage` at the top of your `Fastfile` or by setting the environment variable `FASTLANE_OPT_OUT_USAGE`. [Check out the metrics code on GitHub](https://github.com/fastlane/fastlane/tree/master/fastlane_core/lib/fastlane_core/analytics)
+You can easily opt-out of metrics collection by adding `opt_out_usage` at the top of your `Fast file` or by setting the environment variable `FASTLANE_OPT_OUT_USAGE`. [Check out the metrics code on GitHub](https://github.com/fastlane/fastlane/tree/master/fastlane_core/lib/fastlane_core/analytics)
```
Comment (att55) [inline README.md]: Why do you insert the space?

File: VISION.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 _fastlane_ automates the beta and release deployment process for your iOS or Android apps, including build, code signing, automatic screenshot capture, and distribution of your app binaries.
 
-_fastlane_ will continue to evolve in ways that make it indispensable for, and focused on these needs.
+_fastlane_ will continue to evolve in ways that make it indispensable for and focused on these needs.
 
 _fastlane_ aims to be elegant in success, and empathetic in failure.
 
-_fastlane_ provides intelligent defaults for options, prompts for missing information, and a context that automatically shares relevant information between actions. All of this allows a simple, elegant Fastfile to do a lot of powerful work.
+_fastlane_ provides intelligent defaults for options, prompts for missing information, and a context that automatically shares relevant information between actions. All of this allows a simple, elegant Fast file to do a lot of powerful work.
```
Comment (att55) [inline VISION.md]: Why do you insert the space ??

File: README.md (hunk 2 above)
Comment (MarcSteven) [reply README.md]: @att55  Perhaps she thinks the grammar  is not right

**Comment by koppor** [comment]
@att55 Maybe flag this PR as `invalid` as proposed at https://hacktoberfest.digitalocean.com/faq.

**Comment by att55** [comment]
@MarcSteven @koppor Thank you for your comments. I suppose you are probably right.
I think she should have tried to use this tool at least once before created this PR... because the load of maintainers might increase... 💧

File: YourFirstPR.md [hunk 4]
```diff
@@ -39,11 +39,11 @@ When the coding is done and you’re finished testing your changes, you are read
...
 
 _fastlane_ changes a lot and is in constant flux. We usually merge multiple PRs per day, so sometimes when we are done reviewing, your code might not work with the latest master branch anymore. To prevent this, before you make any changes after your code has been reviewed, you should always rebase the latest changes from the master branch.
 
-After your contribution is merged, it’s not immediately available to all users. Your change will be shipped as part of the next release, which is usually once per week. If your change is time critical, please let us know so we can schedule a release for your change.
+After your contribution is merged, it’s not immediately available to all users. Your change will be shipped as part of the next release, which is usually once per week. If your change is time-critical, please let us know so we can schedule a release for your change.
```
Comment (janpio) [inline YourFirstPR.md]: this is a valid change

File: VISION.md [hunk 5]
```diff
@@ -35,12 +35,12 @@ Each _fastlane_ tool has a specific purpose and should be kept focused on the fu
...
 
 Google supports app development teams with the best tools for building, understanding, and growing their mobile app businesses through [Firebase](https://firebase.google.com) and [Fabric](https://get.fabric.io/).
 
 Google has recognized _fastlane_ as the best tool for tackling tough beta and release deployment challenges. The team’s mission is to make _fastlane_ the de facto tool to automate beta deployments and app store releases for iOS and Android apps.
 
-Google intends to keep _fastlane_ open source and available as a standalone tool for users who are not using Firebase or Fabric. Google is also committed to increasing use of _fastlane_ by promoting it through other products and websites, and developing integrations with Firebase and Fabric tools to help them work better together.
+Google intends to keep _fastlane_ open source and available as a standalone tool for users who are not using Firebase or Fabric. Google is also committed to increasing the use of _fastlane_ by promoting it through other products and websites, and developing integrations with Firebase and Fabric tools to help them work better together.
```
Comment (janpio) [inline VISION.md]: this is a valid change

File: VISION.md [hunk 6]
```diff
@@ -35,12 +35,12 @@ Each _fastlane_ tool has a specific purpose and should be kept focused on the fu
 * [_gym_](https://github.com/fastlane/fastlane/tree/master/gym): Building your iOS apps has never been easier
 * [_match_](https://github.com/fastlane/fastlane/tree/master/match): Easily sync your certificates and profiles across your team
 * [_scan_](https://github.com/fastlane/fastlane/tree/master/scan): The easiest way to run tests for your iOS and Mac apps
-* [_precheck_](https://github.com/fastlane/fastlane/tree/master/precheck): Check your app using a community driven set of App Store review rules to avoid being rejected
+* [_precheck_](https://github.com/fastlane/fastlane/tree/master/precheck): Check your app using a community-driven set of App Store review rules to avoid being rejected
```
Comment (janpio) [inline VISION.md]: this is a valid change

File: ToolsAndDebugging.md [hunk 7]
```diff
@@ -54,25 +54,25 @@ You might have to restart your Mac for the changes to be applied. To see if it w
...
 
-We’re not using the built-in network tracker of your browser, since we also need a proxy for our local _fastlane_ install, which will be covered in the next section of this document.
+We’re not using the built-in network tracker of your browser since we also need a proxy for our local _fastlane_ install, which will be covered in the next section of this document.
 
 <img src=".assets/ToolingCharlesRequest.png" />
 
 ### Compare the API requests
 
-They key is to do the same action you want to test on both the website, and in _spaceship_, so you can see how the requests are different.
+The key is to do the same action you want to test on both the website and in _spaceship_, so you can see how the requests are different.
```
Comment (janpio) [inline ToolsAndDebugging.md]: this is a valid change

File: RespondingToIssuesAndPullRequests.md [hunk 8]
```diff
@@ -61,29 +61,29 @@ git checkout pr-1234
...
 
 When adding comments to a review, make sure they are
 - *Polite*: Ask the author nicely to make the changes. We want to create an environment where our contributors like working with us and come back to submit more PRs
 - *Constructive*: Don’t just say ‘This is bad’ or ‘I don’t like this’. Point the author in the right direction for changes they need to make to improve the code
-- *Necessary*: It is often too easy to ask for changes on perfectly fine code because of personal opinions. If the change follows our vision, adheres to our style guides and is simple to understand, don’t ask the author to change it! You can always make follow-up on a merged PR with improvements of your own.
+- *Necessary*: It is often too easy to ask for changes on perfectly fine code because of personal opinions. If the change follows our vision, adheres to our style guides and is simple to understand, don’t ask the author to change it! You can always make a follow-up on a merged PR with improvements of your own.
```
Comment (janpio) [inline RespondingToIssuesAndPullRequests.md]: the correct fix here would to remove `make` here

File: RespondingToIssuesAndPullRequests.md [hunk 9]
```diff
@@ -8,26 +8,26 @@ Also, whenever you submit a comment, don’t ask users for their personal inform
...
 | duplicate | Another issue/PR already exists that we think captures the problem/request |
-| feature | The issue represents a request for an enhancement or new feature |
+| feature | The issue represents a request for enhancement or new feature |
 | you can do this | We're not actively looking at solving this issue, but community help would be appreciated |
 | question | Someone is looking for help, but isn't describing a problem with the software |
 
 ### Tool Labels
 
-Each tool has it’s own label, e.g. _fastlane_, `fastlane_core` and _gym_.
+Each tool has its label, e.g. _fastlane_, `fastlane_core` and _gym_.
```
Comment (janpio) [inline RespondingToIssuesAndPullRequests.md]: this is a valid change

**Review by janpio — CHANGES_REQUESTED** [review]
Thanks @ankitasharma88990, I left comments on all the valid changes you suggested - all the others unfortunately depend on a misunderstanding and could not be merged. Please open a new PR that only includes the valid changes, and we will gladly merge it!

**Pull request closed without merging**